src/_pytest/fixtures.py

```diff
@@ -612,7 +612,7 @@ def _factorytraceback(self):
             fs, lineno = getfslineno(factory)
             p = self._pyfuncitem.session.fspath.bestrelpath(fs)
             args = _format_args(factory)
-            lines.append("%s:%d:  def %s%s" % (p, lineno, factory.__name__, args))
+            lines.append("%s:%d:  def %s%s" % (p, lineno + 1, factory.__name__, args))
```

Comment:
this is incorrect for multi-decorated things and in python3.8 (where the ast now points us at the function itself)

Reply:
The test works for me with Python 3.8.0a2+ also.
Was this changed in the last days only?

Have to check multi-decorated things, but in general this is not the first time that line numbers are off, because they are zero-based internally..

Reply:
hmmm I was expecting https://github.com/python/cpython/pull/9731 to change the outcome here :thinking:

Reply:
Worth investigating probably, but since tests are passing also with 3.8-dev, I think this is fine for now.